src/test_l1_tvSettings.c

```diff
@@ -7905,6 +8516,9 @@ void test_l1_tvSettings_positive_GetTVSupportedDolbyVisionModes (void)
 	bool IsDolbyModeValid = true;
 	unsigned short sizeReceived = 0;
 	unsigned short sizeReceivedRetry = 0;
+	uint32_t dolbyVisonModeCount = 0;
+	tvDolbyMode_t dolbyVisionModevalue = tvDolbyMode_Dark;
+	char keyValue[UT_KVP_MAX_ELEMENT_SIZE] = { 0 };
```

Comment:
As the test_config_read.h has been removed from the test_l1_tvsettings.c file, the previously used MAX_VIDEO_FORMAT and MAX_DIMMING_MODES need to be defined properly. This causes a compilation error. Please check the compilation part as well.